src/test_typing_extensions.py

```diff
@@ -5351,25 +5379,56 @@ def test_invalid_uses(self):
 
         with self.assertRaisesRegex(
             TypeError,
-            'The last parameter to Concatenate should be a ParamSpec variable',
+            'The last parameter to Concatenate should be a ParamSpec variable or ellipsis',
         ):
             Concatenate[P, T]
 
+        # Cannot construct a Callable with Ellipsis in 3.8 as args must be types
+        if sys.version_info[:2] >= (3, 9, 2):
```

Comment:
```suggestion
        if sys.version_info >= (3, 9, 2):
```

Reply:
Tried to keep it consistent with the other locations as I've seen is quite often. In this case good catch.

Reply:
Ah I see the behavior of sys.version_info was changed and hat to be sliced before.